src/aws-documentation-mcp-server/uv.lock

```diff
@@ -61,11 +61,11 @@ dev = [
 
 [package.metadata]
 requires-dist = [
-    { name = "beautifulsoup4", specifier = ">=4.12.0" },
-    { name = "httpx", specifier = ">=0.27.0" },
-    { name = "loguru", specifier = ">=0.7.0" },
-    { name = "markdownify", specifier = ">=1.1.0" },
-    { name = "mcp", extras = ["cli"], specifier = ">=1.6.0" },
+    { name = "beautifulsoup4", specifier = "==4.13.3" },
+    { name = "httpx", specifier = "==0.28.1" },
+    { name = "loguru", specifier = "==0.7.3" },
+    { name = "markdownify", specifier = "==1.1.0" },
+    { name = "mcp", extras = ["cli"], specifier = "==1.6.0" },
```

Comment:
Diving deep into what the effects of dependabot pinning the [`requires-dist`](https://packaging.python.org/en/latest/specifications/core-metadata/#requires-dist-multiple-use) and if this will cause issues for users.

Testing uv's [`versioning-strategy`](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#versioning-strategy--) to change the broaden the specifications.